schema.prisma

```diff
@@ -177,10 +177,20 @@ model Campaign {
   incomingTransfers Transfer[]    @relation("target_campaign")
   vaults            Vault[]
   withdrawals       Withdrawal[]
+  campaignFiles     CampaignFile[]
 
   @@map("campaigns")
 }
 
+model CampaignFile {
+  id         String   @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
+  filename   String   @db.VarChar(200)
+  campaignId String   @map("campaign_id") @db.Uuid
+  campaign   Campaign @relation(fields: [campaignId], references: [id])
+
+  @@map("campaign_files")
+}
```

Comment:
wondering if we need an uploadedBy: Person relation if not only the coordinator can upload a file.
do we also need the createdAt field or we will take that from S3?

Reply:
I added that and is now retrieved from the auth token. Meaning that we can restrict the access to that endpoint based on roles in the controller but the business logic will always work.